pkg/app/kopia_repository_server_mysql.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2023 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/helm"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/poll"
+)
+
+type KopiaRepositoryServerMysqlDB struct {
+	cli       kubernetes.Interface
+	namespace string
+	name      string
+	chart     helm.ChartInfo
+}
+
+var _ HelmApp = &KopiaRepositoryServerMysqlDB{}
+
+// NewKopiaRepositoryServerMysqlDB Last tested working version "6.14.11"
+func NewKopiaRepositoryServerMysqlDB(name string) HelmApp {
+	return &KopiaRepositoryServerMysqlDB{
+		name: name,
+		chart: helm.ChartInfo{
+			Release:  appendRandString(name),
+			RepoURL:  helm.BitnamiRepoURL,
+			Chart:    "mysql",
+			RepoName: helm.BitnamiRepoName,
+			Values: map[string]string{
+				"auth.rootPassword": "mysecretpassword",
+				"image.pullPolicy":  "Always",
+			},
+		},
+	}
+}
+
+func (mdb *KopiaRepositoryServerMysqlDB) Chart() *helm.ChartInfo {
+	return &mdb.chart
```

Comment:
Is retiring a pointer to the internal field desired? This might lead to external changes that could be potentially undesired here. WDYT?